src/Features/CSharp/Portable/CodeFixes/Async/CSharpAddAwaitCodeFixProvider.cs

```diff
@@ -76,8 +72,19 @@ private bool IsCorrectReturnType(ExpressionSyntax expression, SemanticModel sema
 
         private static ExpressionSyntax ConvertToAwaitExpression(ExpressionSyntax expression)
         {
-            return SyntaxFactory.AwaitExpression(expression)
-                                .WithAdditionalAnnotations(Formatter.Annotation);
+            AwaitExpressionSyntax result;
+
+            if (expression.HasLeadingTrivia)
```

Comment:
@jmarolf I created a standalone CodeAnalysis project, placed a breakpoint on the closing curly brace and ran:
```C#
        static void Main( string[] args )
        {
            var tree = CSharpSyntaxTree.ParseText(@"
using System;
using System.Threading.Task;

class Program
{
    async Task<int> A()
    {
        return true ? Task.FromResult(0) /*0*/ : Task.FromResult(1) /*1*/;
    }
}
");
            var root = tree.GetRoot();
            var cond = (ConditionalExpressionSyntax)root.DescendantNodes().First(n => n is ConditionalExpressionSyntax);
            var paren = SyntaxFactory.ParenthesizedExpression(cond).WithTriviaFrom(cond);
        }
```

The syntax on paren looks like `(true ? Task.FromResult(0) /*0*/ : Task.FromResult(1) /*1*/) /*1*/`, I don't think `WithTriviaFrom` did the override.